src/facade/dragonfly_connection.cc

```diff
@@ -542,7 +525,44 @@ void Connection::ConnectionFlow(FiberSocketBase* peer) {
   --stats_->num_conns;
 }
 
-auto Connection::ParseRedis() -> ParserStatus {
+void Connection::DispatchCommand(uint32_t consumed, mi_heap_t* heap) {
+  bool can_dispatch_sync = (consumed >= io_buf_.InputLen());
```

Comment:
I took it from the previous version. It looks to me like this should actually be a DCHECK. How could we dispatch async if we didn't read fully 🤔